src/components/InputV2/Select.tsx

```diff
@@ -51,19 +51,17 @@ export const Select = ({
   onEnter,
   setIsValid
 }: SelectProps) => {
-  // Make sure value is always a valid option or
-  // an empty string when there is a placeholder
-  useEffect(() => {
-    if (!options.includes(value)) {
-      const defaultValue = placeholder ? '' : options[0]
-      if (value !== defaultValue) onChange(defaultValue)
-    }
-  }, [value, placeholder, options, onChange])
+  // Automatically select the first option if value
+  // is not set and there is no placeholder text
+  useEffect(
+    () => !value && !placeholder && options[0] && onChange(options[0]),
+    [value, placeholder, options, onChange]
+  )
 
   // Communicate validity to parent component
   useEffect(
-    () => setIsValid(!required || !!value),
-    [value, required, setIsValid]
+    () => setIsValid(value ? options.includes(value) : !required),
+    [value, options, required, setIsValid]
```

Comment:
@Schwartz10 This will flag the input as invalid if value is not present in the options, which can happen when an option is selected but the list of options is still populating